AK/StringView.cpp

```diff
@@ -75,6 +75,32 @@ Vector<StringView> StringView::split_view(const char separator, bool keep_empty)
     return v;
 }
 
+Vector<StringView> StringView::split_view(const StringView& separator, bool keep_empty) const
+{
+    if (is_empty())
+        return {};
+
+    ASSERT(!separator.is_empty());
+
+    StringView view { *this };
+
+    Vector<StringView> parts;
+
+    auto maybe_separator_index = find_first_of(separator);
+    while (maybe_separator_index.has_value()) {
+        auto separator_offset = maybe_separator_index.value();
```

Comment:
`offset` vs `index`

Reply:
Ah, that made sense before changing the view itself, not so much now